Sources/Props/ProgressView/BarsProgressStyle.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  BarsProgressStyle.swift
+//  
+//
+//  Created by Djuro on 1/24/22.
+//
+
+import SwiftUI
+
+public struct BarsProgressStyle: ProgressViewStyle {
+    
+    // MARK: - Initializer
+    public init() {}
+    
+    // MARK: - ProgressViewStyle
+    public func makeBody(configuration: Configuration) -> some View {
+        BarsIndicatorView(count: 8,
+                          spacing: 8,
+                          cornerRadius: 8,
+                          scaleRange: 0.5...1,
+                          opacityRange: 0.25...1)
+            .frame(width: 200, height: 50)
+    }
+    
+    struct BarsIndicatorView: View {
```

Comment:
Indicator is a weird name. Since it's effectively a private View, you could follow the path the other progress view styles used and just call it `IndeterminateView` as that clearly describes what it's doing and prevents having to come up with another name. :D